<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWindow</class>
 <widget class="QMainWindow" name="ClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="chat_edit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="message_edit"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="button_layout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="sendType_combo">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimumContentsLength">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>UDP message</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TCP message</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TCP file</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="send_pb">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="generate_layout">
      <item>
       <widget class="QLabel" name="minWordLength_label">
        <property name="text">
         <string>Min Word Length: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="minWordLength_spin">
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="maxWordLength_label">
        <property name="text">
         <string>Max Word Length: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="maxWordLength_spin">
        <property name="value">
         <number>7</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="wordCount_label">
        <property name="text">
         <string>Word Count: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="wordCount_spin">
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>75</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="generate_pb">
        <property name="text">
         <string>Generate message</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>619</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
